### PR TITLE
Adding logistic regression changes to bindings

### PR DESCRIPTION
**Pull request by mmerce**

### CODE REVIEW

**Comment by mmerce** [comment]
Merged last two news into one

File: bindings/news.md [hunk 1]
```diff
@@ -1,165 +1,214 @@
...
+detailed information.
 
-The python example for the REST calls for `logisticregression` can be found in
-[commit 1042538](https://github.com/bigmlcom/python/commit/1042538762096b89f71cb9e3f8ac96660b9bca03).
+The basics will be creating wrappers for the
+REST api calls to create, get, update and delete associations. Their
+structure will be similar to the `cluster` CRUD calls. Associations
+have a private resource id and can also be shared.
+
+The information it returns, encloses an `associations` block
```
Comment (jaor) [inline bindings/news.md]: typo: it -> they

File: bindings/news.md [hunk 2]
```diff
@@ -1,165 +1,214 @@
...
-  the field's structure). Any occurrence of a term in `term_forms` must be
-  added to the occurrences of the corresponding main term in the `tag_cloud`.
-- Check the text field value of the field in your input data. If missing, then
-  add to the `lr_exponent` the `mth + 1` coefficient. Otherwise, parse the
-  terms, compare them to the `tag_cloud` and `term_forms` lists, count the
-  occurrences of each and add the corresponding `b<sub>i</sub> * occurences`
-  contribution to `lr_exponent`.
-- Compute the logistic function and normalize across objective field
-  categories, as in the numeric case.
+*Affects:* Its a new object that encapsulates the JSON information downloaded
```
Comment (jaor) [inline bindings/news.md]: typo: it's

**Comment by mmerce** [comment]
Changes in

**Pull request merged**